dpnp/backend/kernels/dpnp_krnl_indexing.cpp

```diff
@@ -175,6 +175,25 @@ void dpnp_fill_diagonal_c(void* array1_in, void* val_in, size_t* shape, const si
     return;
 }
 
+template<typename _DataType>
+void dpnp_place_c(void* arr_in, void* mask_in, void* vals_in, const size_t arr_size, const size_t vals_size)
+{
+    _DataType* arr = reinterpret_cast<_DataType*>(arr_in);
```

Comment:
``` suggestion
    if (!arr_size)
    {
    	return;
    }
    _DataType* arr = reinterpret_cast<_DataType*>(arr_in);
```

Also it is good to check given pointers, that they are not `nullptr`.
Add checks for `arr_in`, `mask_in` and `vals_in`